onnxscript/tests/function_libs/torch_lib/ops_test_common.py

```diff
@@ -79,6 +79,7 @@ def xfail(
     *,
     reason: str,
     dtypes: Optional[Collection[torch.dtype]] = None,
+    matcher: Optional[Callable[[Any], Any]] = None,
```

Comment:
Extend how we use info in xfail so that we actually assert that it fails?

Reply:
Could you give more details? I just thought skipping subtests with reason fixme doesn't feel right, so would like to enable xfail for those cases.

Reply:
https://github.com/microsoft/onnxscript/issues/669